### ISSUE TITLE
Unicode emoji shortcode

### ISSUE THREAD

**Issue by greentore** [ajbura/cinny#1182]
### Describe the bug

Emoji with shortcodes containing unicode characters are currently not supported. When you select such emoji in the emoji picker and send the message the shortcode won't be replaced with an image.

### Reproduction

1. Click an emoji with a unicode shortcode in the emoji picker, for example :どやっ:
2. The shortcode will be pasted to the message box
3. Send the message
4. The message contains :どやっ:

### Expected behavior

The shortcode should be replaced with an image.

### Platform and versions

```shell
OS: Windows 10
Browser: Chromium 111
Cinny version: v2.2.4
```
